### DISCUSSION TITLE
Magsafe led indicator should work according to aldente's settings.

### DISCUSSION THREAD

**Discussion by csabigabi** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#787] [Ideas]
Is it possible to change tha magsafe led when the machine is full (aldente full) ?

**Comment by MatthiasKerbl** [comment]
Hi @csabigabi,

We are working on implementing the MagSafe LED. However, I can not give a timeframe when this will be implemented.

Cheers,
Matthias

**Reply by csabigabi** [reply]
I'm glad you implemented this feature but as a paid one?!
I'm really disappointed. I mean there is a feature (the LED) which doesn't work properly with your software and you asking money to work as intended.
I could understand if you would make the original funcions (orange=charging, green=charged) free, and the rest to paid.

Have a nice day!
